solver.rb

```diff
@@ -10,20 +11,14 @@ def reverse(str)
   end
 
   def fizzbuzz(num)
-    str = '1'
-    i = 2
-    while i <= num
-      str += if (i % 15).zero?
-               ', fizzbuzz'
-             elsif (i % 3).zero?
-               ', fizz'
-             elsif (i % 5).zero?
-               ', buzz'
-             else
-               ", #{i}"
-             end
-      i += 1
-    end
-    str
+    if (num % 15).zero?
+        return 'fizzbuzz'
+    elsif (num % 3).zero?
+        return 'fizz'
+    elsif (num % 5).zero?
+        return 'buzz'
+    else
+        return num       i
```

Comment:
- [ ] please notice that the value `i` is undefined here, and you can use string interpolation to return value of num like `"#{num}"`